ui/components/app/add-network/index.scss

```diff
@@ -51,6 +75,21 @@
       color: var(--color-text-alternative);
     }
   }
+
+  &__edge-case-box {
+    border: 1px solid var(--brand-colors-grey-grey100);
+
+    &__link {
+      color: var(--color-info-default);
+      display: inline !important;
+      padding: 0 !important;
+    }
+
+    &__button {
+      display: inline !important;
+      padding: 0 !important;
```

Comment:
There are a lot of `!important` usages in this file.  Can these be avoided with a parent `className`?

Reply:
Fixed in: dc67e851ff3027e78e329448e64b8f8310629bf8